jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh

```diff
@@ -719,3 +720,26 @@ function configure_server_state() {
 function generate_random_id() {
     cat /dev/urandom | env LC_CTYPE=C tr -dc 'a-zA-Z0-9' | fold -w 16 | head -n 1
 }
+
+function configure_jbpm_cluster(){
+    if [ ${KIE_JBPM_CLUSTER} = 'true' ]; then
+      configure_jbpm_cache
+      log_info "KIE_JBPM_CLUSTER enabled"
+    else
+      log_info "KIE_JBPM_CLUSTER disabled"
+    fi
+}
+
+function configure_jbpm_cache() {
+  local startTag='<cache-container name="server" aliases="singleton cluster" default-cache="default" module="org.wildfly.clustering.server">'
+  local jbpmTag="<cache-container name='jbpm'>\
+        <transport lock-timeout='60000'/>\
+        <replicated-cache name='nodes'>\
+        <transaction mode='BATCH'/>\
+        </replicated-cache>\
+        <replicated-cache name='jobs'>\
+        <transaction mode='BATCH'/>\
+        </replicated-cache>\
+        </cache-container>"
+    sed -i "s#${startTag}#${jbpmTag}\n${startTag}#g" ${JBOSS_HOME}/standalone/configuration/standalone-openshift.xml
```

Comment:
user the env ${CONFIG_FILE} instead to use the full conf path.

Reply:
ok

Reply:
ok

Reply:
ok

Reply:
it is set here: https://github.com/jboss-container-images/jboss-kie-modules/blob/master/jboss-kie-kieserver/added/openshift-launch.sh#L15

should be available for all functions called after that.

Reply:
See here a failed with ${CONFIG_FILE} https://github.com/jboss-container-images/jboss-kie-modules/pull/434/commits/ff6e6b0f35118dfaa278b5751fcf85bb53bc2b1b